kie-server-parent/kie-server-tests/kie-server-integ-tests-controller/src/test/java/org/kie/server/integrationtests/controller/KieControllerManagementIntegrationTest.java

```diff
@@ -726,42 +726,6 @@ public void testUpdateContainerNonValidReleaseId() throws Exception {
     }
 
     @Test
-    public void testStartContainerByUpdateKieContainerStatus() throws Exception {
```

Comment:
@karreiro What is the reason for deleting of this test? The test checks that it is possible to change container status by container spec update. When container status is changed to STARTED then controller should start container on Kie servers.